### ISSUE TITLE
DeepLabCut[gui] fails to run on Debian10

### ISSUE THREAD

**Issue by schloegl** [DeepLabCut/DeepLabCut#1198]
Thanks for opening this issue, and thanks for using DeepLabCut (we hope you are enjoying it ;). 
Please fill out the template completely, including the full "TRACEBACK" and input code that you ran to hit this error.

**Describe the bug**

We are trying to make DeepLabCut run on Debian10 (buster). We try to do this with and without GPU support, the result is the same. The issue is installing wxpython, trying to install wxPython-4.0.7.post2.tar.gz 
fails with this error 
```

    copying symlink wx/libwx_baseu-3.0.so -> build/lib.linux-x86_64-3.7/wx/libwx_baseu-3.0.so
    error: [Errno 2] No such file or directory: 'build/lib.linux-x86_64-3.7/wx/libwx_baseu-3.0.so'
```

I modifying the setup.py script such that wxPython 4.1 can be installed, will install the GUI-version of deeplabcut, but at runtime it will eventually fail with this error.

```

 widgetsizer.Add(self.quit, 1, wx.ALL | wx.ALIGN_RIGHT, 15)
wx._core.wxAssertionError: C++ assertion "!(flags & wxALIGN_RIGHT)" failed at /home/wxpy/wxPython-4.1.1/ext/wxWidgets/src/common/sizer.cpp(2168) in DoInsert(): Horizontal alignment flags are ignored in horizontal sizers
Segmentation fault
```

Please provide the minimal required code to reproduce the error and the full output (please edit ONLY the section below that says `PLACE YOUR CODE HERE!!! `.

This is the installation procedure: 
```
python3 -m venv $PREFIX
source ${PREFIX}/bin/activate
pip install -U pip setuptools 
pip install deeplabcut 
pip install tensorflow==1.15.5
pip install deeplabcut[gui]	# will fail, but install all requirements except for wxpython 
pip install wxpython==4.1.1

```

**Desktop (please complete the following information about your system):**
 - OS: [e.g. Windows10, MacOS version, Linux version, etc.]

Debian10(buster), Tensorflow-1.15.5 (or Tensorflow-gui-1.15.5, and Cuda 10.0, cudnn 7.4)

 - DeepLabCut Version [e.g. 22] (please check with ``import deeplabcut``, ``deeplabcut.__version__``)
2.1.10.2 and 2.1.10.3 (from git repository)

 - DeepLabCut mode, i.e. single animal tracking, multi-animal tracking, 3D tracking.

I'm only adminstrating the machine, I'd need to check with the user, for these details.  Let me know whether this is important to you. 

4. See error:

<details><summary>TRACEBACK</summary><p>

```python

 widgetsizer.Add(self.quit, 1, wx.ALL | wx.ALIGN_RIGHT, 15)
wx._core.wxAssertionError: C++ assertion "!(flags & wxALIGN_RIGHT)" failed at /home/wxpy/wxPython-4.1.1/ext/wxWidgets/src/common/sizer.cpp(2168) in DoInsert(): Horizontal alignment flags are ignored in horizontal sizers
Segmentation fault

```
</p></details>

**Expected behavior**

The expected behaviour is that DeepLabCut[GUI] can be used with wxPython 4.1.

**Comment by MMathisLab** [comment]
HI @schloegl (hard to follow your post, maybe you can edit such that it;s a bit more in line with the template ;), but: "Debian10 (buster)." --> I don't know how this works with wxPython wheels. you might want to try to install deeplabcut (no gui) and then simply build the wheels: https://wxpython.org/pages/downloads/index.html

**Comment by schloegl** [comment]
Thanks for the response. I cleanup the formatting in order to clarify the issue. Basically, the issue is that wxPython 4.0.7 fails to build on Debian10, and wxPython does not provide wheels for it, probably for the same reason. 

wxPython4.1 can be installed on Debian10, but then DeepLabCut (DLC) eventually fails. It would be great if DLC could be made to work with wxPython 4.1.

**Comment by MMathisLab** [comment]
We don’t have plans to support past 4.0.7 as wxPython will be dropped next quarter in favor of napari; it does build on Ubuntu (that is all we test/use) if that’s an option.

**Comment by schloegl** [comment]
For the sake of completeness, here are the commands for installing deeplabcut on Debian10. 

```
PREFIX=DLC-CPU
python3 -m venv $PREFIX
source ${PREFIX}/bin/activate
pip install -U pip
pip install setuptools==43.0.0
pip install deeplabcut 
pip install tensorflow==1.15.5
pip install wxPython==4.0.7.post2
pip install deeplabcut[gui]        
deactivate 

```
